[2025-06-08T15:14:34.998+0000] {scheduler_job_runner.py:781} ERROR - Executor reports task instance <TaskInstance: bento_api_pipeline.bento_train manual__2025-06-08T15:14:06.603564+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
